src/x/instrument/string_list_emitter.go

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package instrument
+
+import (
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/uber-go/tally"
+)
+
+const (
+	// stringListEmitterWaitInterval defines the time to wait between emitting the value of the Gauge
+	// again.
+	stringListEmitterWaitInterval = 10 * time.Second
+)
+
+var (
+	errStringListEmitterAlreadyRunning = errors.New("can't start StringListEmitter: already running")
+	errStringListEmitterNotStarted     = errors.New("StringListEmitter is not running")
```

Comment:
nit: We try to stay strictly to the Effective Go recommendation of naming all logs and errors with letters beginning in lower case.

Also I usually just name things in separated words, even it relates to a symbol, it's still searchable in the code, etc from a log line.

i.e.
```go
		errStringListEmitterAlreadyRunning = errors.New("string list emitter: already running")
	errStringListEmitterNotStarted     = errors.New("string list emitter: not running")
```

Reply:
Good point, fixed it.